.github/workflows/amazon-cloudwatch-observability-image-scan.yaml

```diff
@@ -0,0 +1,92 @@
+name: Run Image Scan for Amazon CloudWatch Observability Helm Chart
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [ opened, reopened, synchronize, ready_for_review ]
+    branches:
+      - main
```

Comment:
Instead of running this on each PR, can we just have it run on a schedule?

Reply:
changed to cron job to run every 9am est